app/components/Onboarding/Syncing.js

```diff
@@ -8,14 +8,7 @@ import { showNotification } from 'notifications'
 import styles from './Syncing.scss'
 
 class Syncing extends Component {
-  componentWillMount() {
-    const { fetchBlockHeight, blockHeight } = this.props
-
-    // If we don't already know the target block height, fetch it now.
-    if (!blockHeight) {
-      fetchBlockHeight()
-    }
-  }
+  componentWillMount() {}
```

Comment:
I think we can drop this method, right?

Reply:
The reason why I left is because I know we still need it in the the next PR with part of https://github.com/LN-Zap/zap-desktop/pull/538 that is about to follow... Due to our lint rules, removing this method requires us to refactor this class into a pure function, which would then have to be refactored back in the next PR!

I know its not quite nice and clean, but can we just leave this empty method in for this one commit - it won\t be empty by the next commit!